src/viewers/VivViewer.js

```diff
@@ -2,12 +2,18 @@ import React, { PureComponent } from 'react'; // eslint-disable-line import/no-u
 import DeckGL from '@deck.gl/react';
 import { getVivId } from '../views/utils';
 
+/**
+ * @callback ViewStateChange
+ * @param {Object} event
+ */
+
```

Comment:
Can we remove the other two references to this?  It seems like only one is needed for the docs and the docs compiler we use generates multiple references to this.

Reply:
I figured out how to import this definition. That's much better than defining it multiple times. Without the import, VS Code doesn't show the correct type on mouse-over.